Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts

```diff
@@ -17,7 +17,7 @@ export function addArguments(helmCli: helmcli) : void {
         helmCli.addArgument("--dependency-update");
     }
 
-    if(save) {
+    if(save && !helmCli.isHelmV3()) {
```

Comment:
Fix indent here as well

Reply:
Does it save by default for `v2`?

Reply:
Yes. The default value is true for `save`